src/models/historical_location.js

```diff
@@ -32,7 +32,8 @@ module.exports = (sequelize, DataTypes) => {
     time: { type: DataTypes.DATE, allowNull: false }
   });
 
-  // XXX relations
+  // Relations with Location and Thing are already specified on
```

Comment:
Is it worthy to bidirectionaly specify these associations? I guess that would allow us to do `HistorlicalLocation.findById(:id).Things` when requesting `/HistoricalLocations(:id)`.